static/bidder-info/insticator.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint: "https://ex.ingage.tech/v1/prebidserver"
+maintainer:
+  email: "predid@insticator.com"
```

Comment:
Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread